demo_data/.well-known/product-passport/products/1V3907.json

```diff
@@ -0,0 +1,23 @@
+{
+  "names": {
+    "en_US": "BARTO CHEF'S JACKET"
+  },
+  "image_url": "https://tags.ioxio.dev/images/chef-jacket.jpg",
+  "supported_dataproducts": [
+    {
+      "path": "DigitalProductPassport/Textile/Garment/ProductDataSheet_v0.1",
+      "source": "dpp_demo"
+    },
+    {
+      "path": "DigitalProductPassport/Textile/Garment/MaterialDisclosureSheet_v0.1",
+      "source": "dpp_demo"
+    },
+    { "path": "DigitalProductPassport/Textile/Garment/MaintenanceLog_v0.1", "source": "dpp_demo" },
+    { "path": "DigitalProductPassport/Textile/Garment/BillofMaterials_v0.1", "source": "dpp_demo" },
```

Comment:
To my understanding the `MaintenanceLog` and `BillOfMaterials` (note capital `O`, not lower case) will not be used for either one of the two products, but for an upcoming other product. All definitions were still requested to be created already.
```suggestion
```